### ISSUE TITLE
Fix accessibility issues from Lighthouse audit

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#4067]
## Lighthouse results

- [ ] [id] attributes on the page are not unique - The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies.

```
Failing Elements
--
rect#backgroundRect
g#maxExtent
g#bars
g#groups
```

- [x] Lists do not contain only `<li>` elements and script supporting elements (`<script>` and `<template>`). Screen readers have a specific way of announcing lists. Ensuring proper list structure aids screen reader output.

```
Failing Elements
--
ul.header-nav_top
```

- [x] `<html>` element does not have a `[lang]` attribute - If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
```
Failing Elements
--
html
```

- [ ] Form elements do not have associated labels - Labels ensure that form controls are announced properly by assistive technologies, like screen readers.

```
Failing Elements
--
select
select
<select><option value="calendar_year">Calendar year</option><option selected="" value="fiscal_year">Fiscal year</option></select>
```
